ipyfilechooser/utils.py

```diff
@@ -3,9 +3,10 @@
 import os
 import string
 import sys
+from typing import Union, List, AnyStr, Sequence, Iterable, Optional
 
 
-def get_subpaths(path):
+def get_subpaths(path: Union[str, bytes, os.PathLike[str], os.PathLike[bytes]]) -> List[Union[str, bytes, os.PathLike[str], os.PathLike[bytes]]]:
```

Comment:
Why is the typing hint for this function different from the `AnyStr` type hint in the `has_parent` function below? Input for `get_subpaths` is a string, output is a list of strings.